modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -176,11 +176,15 @@ struct IEUnit {
 
     explicit IEUnit(const cv::gapi::ie::detail::ParamDesc &pp)
         : params(pp) {
-
+#if INF_ENGINE_RELEASE < 2020000000  // < 2020.1
         IE::CNNNetReader reader;
         reader.ReadNetwork(params.model_path);
         reader.ReadWeights(params.weights_path);
         net = reader.getNetwork();
+#else // >= 2020.1
+        IE::Core core;
```

Comment:
Core shoud be a single tone object - otherwise you will face NC_Error in case of Myriad target for second network

Reply:
Made Core singleton